app/renderer/js/utils/system-util.ts

```diff
@@ -28,22 +33,23 @@ class SystemUtil {
 	}
 
 	getOS() {
-		if (os.platform() === 'darwin') {
+		const platform: string = os.platform();
+		if (platform === 'darwin') {
 			return 'Mac';
-		}
-		if (os.platform() === 'linux') {
+		} else if (platform === 'linux') {
 			return 'Linux';
-		}
-		if (os.platform() === 'win32' || os.platform() === 'win64') {
+		} else if (platform === 'win32' || platform === 'win64') {
```

Comment:
Removed the `win64` check

Reply:
win64 doesn't exist! So, was remove therefore. See https://nodejs.org/api/os.html#os_os_platform .